src/ripple/app/paths/RippleCalc.cpp

```diff
@@ -100,17 +114,17 @@ RippleCalc::Output RippleCalc::rippleCalculate (
         }
     }
 
-    j.debug () << "RippleCalc Result> " <<
-            " actualIn: " << flowOut.actualAmountIn <<
-            ", actualOut: " << flowOut.actualAmountOut <<
-            ", result: " << flowOut.result () <<
-            ", dstAmtReq: " << saDstAmountReq <<
-            ", sendMax: " << saMaxAmountReq <<
-            ", algo: " << "V2";
+    j.debug() << "RippleCalc Result> "
+              << " actualIn: " << flowOut.actualAmountIn
+              << ", actualOut: " << flowOut.actualAmountOut
+              << ", result: " << flowOut.result()
+              << ", dstAmtReq: " << saDstAmountReq
+              << ", sendMax: " << saMaxAmountReq << ", algo: "
+              << "V2";
```

Comment:
Is the "algo: V2" still relevant?

Reply:
I have some old scripts that use it, but on re-consideration, those scripts are obsolete now. Removed.